### ISSUE TITLE
Froala v4.2.2 truncating <a> href values upon initial load

### ISSUE THREAD

**Issue by burtonrhodes** [froala/wysiwyg-editor#4815]
When loading an editor with predefined html, the <a> href value is truncated.  The value should be the same as the initial html.

##### Steps to reproduce the problem.
Here is a jsFiddle demonstrating the problem with the latest 4.2.2
https://jsfiddle.net/burtonrhodes/yxdpoLsk/8/

You'll notice the initial text is
```
<p>
<a href="http://localhost:8080/afs/portal/?xactionId=139263">http://localhost:8080/afs/portal/?xactionId=139263</a>
</p>
```
But the editor modifies the html href value to 
```
<p>
<a href="http://localhost:8080/afs/portal/?xacti">http://localhost:8080/afs/portal/?xactionId=139263</a>
</p>
```

##### Editor version.
4.2.2  (this does not happen in version 4.2.1)

##### OS.
Windows 11

##### Browser.
Chrome Version 128.0.6613.120 (Official Build) (64-bit)

**Comment by forum-is** [comment]
We can confirm this issue, also irrevocably breaks "src" attributes on img tags. Please fix ASAP.

**Comment by burtonrhodes** [comment]
@forum-is You might consider emailing support@froala.com and reference this issue as I'm not sure how often this repository is checked.

**Comment by ilyaskarim** [comment]
We are aware of this issue, and it is scheduled for a fix in our next editor release, V4.3.1.

**Comment by richardnorthcottvisit** [comment]
We're seeing the same issues with the URL truncations when used in HREF values. We also use querystring propery names which include the characters 'on' - this appears to be where the truncation begins.

Example
`<a href="https://mysite.com/s?cont=1234">link</a>`

becomes
`<a href="https://mysite.com/s?c">link</a>`

**Comment by ilyaskarim** [comment]
Fixed in V4.3.1: https://froala.com/wysiwyg-editor/changelog/#4.3.1
